src/item.cpp

```diff
@@ -9297,8 +9297,8 @@ std::string item::variant_description() const
 
     // append the description instead of fully overwriting it
     if( itype_variant().append ) {
-        return _( string_format( "%s  %s", type->description.translated(),
-                                 itype_variant().alt_description.translated() ) );
+        return string_format( "%s  %s", type->description.translated(),
```

Comment:
```suggestion
        return string_format( pgettext( "variant description", "%s  %s" ), type->description.translated(),
```
Some languages might want to remove spaces between two sentences.